bears/rust/RustLintBear.py

```diff
@@ -0,0 +1,26 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='rustc',
+        output_format='regex',
+        use_stderr=True,
+        prerequisite_check_command=('rustc', '--version'),
+        prerequisite_check_fail_message='rustc is not installed.',
+        output_regex=r'(?P<file>.+?):(?P<line>\d+):(?P<col>\d+):\s+\d+:\d+\s'
+                     r'(?:(?P<error>(error|fatal error))|'
+                     r'(?P<warning>warning)):\s+(?P<message>.+)')
+class RustLintBear:
+    """
+    This bear checks ``rust`` code for syntax and formatting issues using
+    ``rustc`` utility.
```

Comment:
https://www.rust-lang.org/en-US/downloads.html

`rustc -Zno-trans` This argument is not even present in help section of command line. Found it on https://github.com/oschwald/SublimeLinter-contrib-rustc/blob/master/linter.py#L25